scripts/ci/libraries/_runs.sh

```diff
@@ -83,3 +83,13 @@ function runs::run_prepare_provider_documentation() {
         "${AIRFLOW_CI_IMAGE_WITH_TAG}" \
         "--" "/opt/airflow/scripts/in_container/run_prepare_provider_documentation.sh" "${@}"
 }
+
+function runs::run_db_upgrade_and_downgrade() {
+    docker_v run "${EXTRA_DOCKER_FLAGS[@]}" \
+        --entrypoint "/usr/local/bin/dumb-init"  \
+        -t \
```

Comment:
Do we need a tty?

Reply:
I'm changing this implementation cause it doesn't use the underlying DB